mlrun/config.py

```diff
@@ -104,7 +104,6 @@
     # FIXME: Adding these defaults here so we won't need to patch the "installing component" (provazio-controller) to
     #  configure this values on field systems, for newer system this will be configured correctly
     "v3io_api": "http://v3io-webapi:8081",
-    "redis_url": "redis://localhost:6379",
```

Comment:
Doesn't it make sense to still have a default url? For example, if I want to use the default I can still use something like `redis:///my/path` (which will use the default endpoint here), but if I want to specify a different redis host I can use something like `redis://myhost:6379/my/path` that will go to the other host. 
I actually thought this is how this works before this PR - isn't it the case?

Reply:
I agree, I think we should still have default `redis_url` so that if user works only with one redis it won't have to define path each time. 

Also I have two comments: 
1. Something that we experienced multiple times is that we first define one attribute related to some feature/resource and then after time passed some new use cases appear which requires adding more configuration attributes related to the same resource, so I would suggest defining root attribute `redis` e.g : 
```
"redis": {
  "url": "some-path",
}
``` 
I see we also have `"datastore"` root attribute from type dict so maybe add `redis` inside.

2. I am not sure if that is the case here, but if the expected behavior is that the clients will write to the redis and not only the API this means  `redis_url` param is supposed to be passed to each mlrun client running, the way we are doing it is by adding the params required for clients inside the `ClientSpec` class which is being requested and loaded on each mlrun client initialization from the API.

Reply:
I agree, I think we should still have default `redis_url` so that if user works only with one redis it won't have to define path each time. 

Also I have two comments: 
1. Something that we experienced multiple times is that we first define one attribute related to some feature/resource and then after time passed some new use cases appear which requires adding more configuration attributes related to the same resource, so I would suggest defining root attribute `redis` e.g : 
```
"redis": {
  "url": "some-path",
}
``` 
I see we also have `"datastore"` root attribute from type dict so maybe add `redis` inside.

2. I am not sure if that is the case here, but if the expected behavior is that the clients will write to the redis and not only the API this means  `redis_url` param is supposed to be passed to each mlrun client running, the way we are doing it is by adding the params required for clients inside the `ClientSpec` class which is being requested and loaded on each mlrun client initialization from the API.

Reply:
Saar, Tom - I'm fine with your suggestion to keep the redis-url (as is or inside a root attribute).
And indeed, I assume I will need to address forwarding the redis_url config to all clients.
I would like keep this PR for addressing multiple redis servers from the url (so the feature can be tested)
And on a following PR add the redis_url properly as discussed.